### ISSUE TITLE
[Android] Thumbnailbar is rendered on top of the pdf 

### ISSUE THREAD

**Issue by leoek** [PSPDFKit/react-native#273]
# Details

After the fix for #252 `PSPDFKitView` applies the new configuration if it was changed. However the thumbnailbar is rendered on top of the pdf if it isn't enabled from the first render on. I would expect that the thumbnailbar renders below the pdf like it does with a static configuration.

Current:

![2019-08-07-210526_571x563_scrot](https://user-images.githubusercontent.com/9424038/63606035-7e565500-c5cf-11e9-9987-d47f7fab8255.png)

Expected:

![2019-08-07-210711_636x424_scrot](https://user-images.githubusercontent.com/9424038/63606044-84e4cc80-c5cf-11e9-94a5-b8f8c1d33a27.png)

Steps To Reproduce:

1. clone https://github.com/leoek/react-native-pspdfkit-demo
2. checkout reproduce-issue-thumbnailbar-position
3. yarn
4. yarn android
5. Press "Go To PdfScreen"
6. Press "thumbnails" (top left)
--> The thumbnailbar is rendered on top of the pdf

# Acceptance Criteria

- [ ] The thumbnailbar is rendered below the pdf and the position of the thumbnailbar is consistent.
  - [ ] Fixed in the current version
  - [ ] Fixed in the last windows compatible version

**Comment by leoek** [comment]
@irgendeinich any news on this?

**Comment by irgendeinich** [comment]
@leoek 
I checked your reproduction and the bug does indeed happen there but I was not able to reproduce this on the latest master. If you are able to reproduce this with the latest release I can take another look.
![thumbnail_bug](https://user-images.githubusercontent.com/2066444/64002810-3d989780-cb0b-11e9-97ce-7408bca85be4.gif)

**Comment by leoek** [comment]
@irgendeinich Unfortunately I cannot upgrade rn past 0.59, therefore react-native-pspdfkit 1.24.9 is the latest master for me.

**Comment by steipete** [comment]
Can you split usage, use master for iOS+Android and legacy for Windows? We can't possibly backport all the things to the older version. Alternative is to wait until Windows has catched up, or going to market with iOS+Android for now and Windows once Microsoft is ready.

**Comment by leoek** [comment]
>Alternative is to wait until Windows has catched up, or going to market with iOS+Android for now and Windows once Microsoft is ready.

Going to market with IOS+Android first is not an option for us, as it is our primary platform.

>Can you split usage, use master for iOS+Android and legacy for Windows?

I've two major concerns with this:
1. This would require us to maintain two codebases until the issue with Windows is resolved. (And we may end up in a scenario with android/IOS always being ahead)
2. The api of pspdfkit/rn-pspdfkit is already inconsistent within one version and our integration contains many workarounds to somehow import the saved annotations across platforms. (see #275, #276, #279, #281, #283, #286 ...) This will be even worse if we have annotations which have to be imported across platforms and versions.

>We can't possibly backport all the things to the older version.

I guess the issue is that you are only reproducing and fixing issues in the latest version. It would be much better if issues were fixed in the version they are reported with and the fix would then be merged into the following versions. This approach is obviously not needed for features but usually a must for bugs. (In a perfect world you'd never need to merge a fix into multiple versions if there are no bugs in your solution.)

Maybe it would help if you'd separate the releases into release candidates which won't receive bug fixes and releases which will receive bug fixes at least until the next release version is ready and tested on all platforms.
Imo 1.24.9 should be a release version as it's the last which supports RN 0.59. RN is not only a requirement for teams using react-native-windows as there were major changes from RN 0.59 and ExpoKit and a lot of libraries are not compatible yet.

After all this wouldn't be an issue if version 1.24.9 (and android 5.0.1) would be stable enough to use in a production environment.

**Comment by steipete** [comment]
>This would require us to maintain two codebases until the issue with Windows is resolved. (And we may end up in a scenario with android/IOS always being ahead)

Yes. This is a risk. There's commitment from Microsoft and they're re-investing quite a lot, so I see this as small risk, but for now this is the only realistic option.

As a general support and product policy we release product updates every 2-6 weeks. We do not offer backports, except in very special cases when our customers pay extra for it. I understand that this would be a preferred solution, but we are a small team and need to optimize for the majority of our users. Apologies if this causes a bit of extra work on your side - we try to help as best as we can with the open issues. If we would also follow a strict backport-policy, it would take twice as long to resolve open issues.

It's unfortunate that Expo(Kit) doesn't support 0.60 yet, based on their forum entries they are already working on it, so it shouldn't be long.
